packages/compiler-cli/test/ngcc/BUILD.bazel

```diff
@@ -18,7 +19,8 @@ jasmine_node_test(
     name = "ngcc",
     bootstrap = ["angular/tools/testing/init_node_no_angular_spec.js"],
     data = [
-        "//packages/compiler-cli/test/ngcc/fake_core:npm_package",
+      "//packages/core:npm_package",
+      "//packages/common:npm_package",
```

Comment:
@alxhub - including these dependencies makes the incremental much slower as they appear to be somewhat rebuilt every time... in particular the Angular Compiler seems to be running on them every build. This means that ~800 targets are built rather than ~13.

Reply:
<img width="958" alt="screen shot 2018-06-19 at 14 54 37" src="https://user-images.githubusercontent.com/15655/41601578-bd71b164-73d0-11e8-8712-91c0ff020421.png">

Reply:
<img width="726" alt="screen shot 2018-06-19 at 14 55 56" src="https://user-images.githubusercontent.com/15655/41601650-e76171ee-73d0-11e8-87db-4d911000e7ca.png">